exasol_integration_test_docker_environment/test/test_test_environment_db_mem_size.py

```diff
@@ -39,16 +41,16 @@ def assert_mem_size(self, size: str):
     def test_default_db_mem_size(self):
         self.docker_environment_name = "test_default_db_mem_size"
         docker_envs = self.test_environment.spawn_docker_test_environment(self.docker_environment_name)
-        self.assert_mem_size("2 GiB")
-        close_environments(*docker_envs)
+        with ContextSpawnedTestEnvironment(*docker_envs):
+            self.assert_mem_size("2 GiB")
 
     def test_smallest_valid_db_mem_size(self):
         self.docker_environment_name = "test_smallest_valid_db_mem_size"
         docker_envs = \
             self.test_environment.spawn_docker_test_environment(self.docker_environment_name,
                                                                 ["--db-mem-size", "'1 GiB'"])
-        self.assert_mem_size("1 GiB")
-        close_environments(*docker_envs)
+        with ContextSpawnedTestEnvironment(*docker_envs):
```

Comment:
Usually, you try to generate the object for the context directly in the with statement